nemo_curator/utils/fuzzy_dedup_utils/io_utils.py

```diff
@@ -26,31 +27,41 @@
 # TODO:
 # Combine this with
 # nemo_curator.distributed_utils.read_cudf_jsonl
-def read_json_func(files, engine="cudf", include_path_column=False, columns=None):
+def _read_json_func(
+    files, engine="cudf", include_path_column=False, columns=None, input_meta=None
+):
     """
     Reads multiple Json Lines files into a cuDF
     dataframe with an additional `path` column denoting the path
     of the input file.
     """
+    if input_meta:
+        read_kwargs["dtype"] = (
```

Comment:
This file also doesn't use `read_kwargs`, probably want to update `input_meta`

Reply:
Great catch! Thank you!

I have actually changed a bit the condition. Because the interface expects either a dictionary, a string or None, I only the string needs to be converted to dictionary, I have changed the logic a bit as follows:

>     if type(input_meta) == str:
>         input_meta = ast.literal_eval(input_meta)